### ISSUE TITLE
iOS build_apple.py 脚本无法编译

### ISSUE THREAD

**Issue by AllenZhangl** [Tencent/mars#411]
我看comm 目录下没有build/comm.build/Release-iphoneos/comm.build/Objects-normal 

![image](https://user-images.githubusercontent.com/34906426/40912963-a8fcbc6c-6825-11e8-9f26-fc21d6a7a491.png)

**Comment by garryyan** [comment]
手动删除 comm的build目录，然后再试一下

**Comment by LuckZXF** [comment]
同样的问题 以前还可以用这个脚本得到xlog相关的framework，现在运行就报上图的错误。

**Comment by hackjie** [comment]
met the same problem in Xcode 9

```
**************merging libs, wait for a monment!!!***********************
merging comm...........
Traceback (most recent call last):
  File "build_apple.py", line 205, in <module>
    main()
  File "build_apple.py", line 190, in main
    build_apple(APPLE_PROJECTS[int(num)-1], save_path)
  File "build_apple.py", line 141, in build_apple
    cpu_folders = os.listdir(obj_folders)
OSError: [Errno 2] No such file or directory: '/Users/leoli/Desktop/mars/mars/libraries/../comm/build/comm.build/Release-iphoneos/comm.build/Objects-normal'
```

**Comment by zsrr** [comment]
最新的提交解决了这个问题，不过前提是xcodebuild指令可以正常使用

**Comment by garryyan** [comment]
#446 #457 #458 
fixed in e8b5773bd68b3c8bb1d27181a36a54e26d7117a7

**Comment by xiaoqingmail** [comment]
Traceback (most recent call last):
  File "./mars/build_ios.py", line 159, in <module>
    main()
  File "./mars/build_ios.py", line 144, in main
    build_ios()
  File "./mars/build_ios.py", line 22, in build_ios
    gen_mars_revision_file('comm', tag)
  File "/Users/lixiaoqing/Downloads/mars-master/mars/mars_utils.py", line 328, in gen_mars_revision_file
    revision, path, url = parse_as_git(version_file_path)
  File "/Users/lixiaoqing/Downloads/mars-master/mars/mars_utils.py", line 319, in parse_as_git
    os.chdir(path)
OSError: [Errno 2] No such file or directory: 'comm'
